packages/@aws-cdk/aws-lambda-nodejs/lib/bundling.ts

```diff
@@ -201,13 +201,15 @@ export class Bundling {
     return lambda.Code.fromAsset(projectRoot, {
       assetHashType: cdk.AssetHashType.BUNDLE,
       bundling: {
-        image,
-        command: ['bash', '-c', chain([parcelCommand, depsCommand])],
-        environment: options.parcelEnvironment,
-        volumes: options.cacheDir
-          ? [{ containerPath: '/parcel-cache', hostPath: options.cacheDir }]
-          : [],
-        workingDirectory: path.dirname(containerEntryPath).replace(/\\/g, '/'), // Always use POSIX paths in the container
```

Comment:
where is the local stuff?

Reply:
not there yet, was planning to separate this to another PR.

Reply:
ok, sounds good

Reply:
just make sure it is possible and we didn't forget anything...